webapp/graphite/render/glyph.py

```diff
@@ -126,6 +126,19 @@
     ('G', 1000.0**3),
     ('M', 1000.0**2),
     ('K', 1000.0   )),
+  'sec': (
+    ('Y', 60*60*24*365),
+    ('M', 60*60*24*30),
+    ('D', 60*60*24),
+    ('H', 60*60),
+    ('m', 60)),
+  'msec': (
+    ('Y', 60*60*24*365*1000),
+    ('M', 60*60*24*30*1000),
```

Comment:
Probably a bit, But since we're talking relative times on the y axis it 
should be good enough. I guess it could use weeks rather than months, 
unless you have an alternate suggestion..

On 15-07-08 11:10 PM, Jeff Schroeder wrote:
>
> In webapp/graphite/render/glyph.py 
> <https://github.com/graphite-project/graphite-web/pull/1220#discussion_r34224084>:
>
> > @@ -126,6 +126,19 @@
> >      ('G', 1000.0**3),
> >      ('M', 1000.0**2),
> >      ('K', 1000.0   )),
> > +  'sec': (
> > +    ('Y', 60*60*24*365),
> > +    ('M', 60*60*24*30),
> > +    ('D', 60*60*24),
> > +    ('H', 60*60),
> > +    ('m', 60)),
> > +  'msec': (
> > +    ('Y', 60*60*24*365*1000),
> > +    ('M', 60*60*24*30*1000),
>
> Isn't that kind of cheating? Not all months are 30 days.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/graphite-project/graphite-web/pull/1220/files#r34224084>.
>